docs/guidelines.md

```diff
@@ -0,0 +1,9 @@
+This issue intends to layout some guidelines on the transfer of [conda-recipes]( https://github.com/conda/conda-recipes ) and [anaconda-recipes]( https://github.com/ContinuumIO/anaconda-recipes ) to conda-forge. These aren't hard and fast rules. They are certainly up for discussion. However, it would be good to come up with a consensus going forward.
+
+Eventually all recipes from those repos should be proposed for addition here. It may be decided that a few don't actually belong or should not be supported anymore. For instance, a `vagrant` package seems a bit silly to me.
```

Comment:
> For instance, a `vagrant` package seems a bit silly to me.

I guess if we are starting to turn this into docs, we should probably drop this sentence.